src/test/java/io/strimzi/kafka/bridge/amqp/AmqpBridgeTest.java

```diff
@@ -109,6 +109,13 @@ public void sendSimpleMessages(TestContext context) {
 		String topic = "sendSimpleMessages";
 		kafkaCluster.createTopic(topic, 1, 1);
 
+		try {
```

Comment:
why the need for sleep now? The tests were all passing up to now.

Reply:
I turned off the autocreation of the topics to test non-existing topics. Before, it just created a topic when a message was sent to it.